pkg/util/log/log.go

```diff
@@ -538,9 +538,20 @@ func UnregisterAdditionalLogger(n string) error {
 	return errors.New("cannot unregister: logger not initialized")
 }
 
+// GetCurrentLogLevel returns a seelog native representation of the current
+// log level
+func GetCurrentLogLevel() (seelog.LogLevel, error) {
+	if logger != nil && logger.inner != nil {
```

Comment:
I think we should `RLock` the logger here, similarly to https://github.com/DataDog/datadog-agent/blob/00636535c44f9d3c69d1962f87ef99c8c3dc5b0d/pkg/util/log/log.go#L104-L108.